src/ebay-education-notice/README.md

```diff
@@ -0,0 +1,61 @@
+# EbayEducationNotice
+
+## Demo
+
+[Storybook](https://opensource.ebay.com/ebayui-core-react/main/?path=/docs/notices-tips-ebay-education-notice--docs)
+
+## Import JS
+
+```jsx harmony
+import {
+    EbayEducationNotice,
+    EbayNoticeContent,
+    EbayEducationNoticeTitle,
+    EbayEducationNoticeFooter
+} from '@ebay/ui-core-react/ebay-education-notice'
+```
+
+### Import following styles from SKIN
+
+```jsx harmony
+import '@ebay/skin/education-notice'
+```
+
+## Use
+
+```jsx harmony
+<EbayEducationNotice
+    educationIcon={<EbayIcon name="theEbayVault24" />}
```

Comment:
Education icon should be just a string. It should be typed with Icon name so it can be autocompleted